### ISSUE TITLE
Awesome job !

### ISSUE THREAD

**Issue by tchaton** [luca-medeiros/lang-segment-anything#1]
Hey @luca-medeiros 

I just had a read to your codebase. Very neat work !

Best,
T.C

**Comment by joe85** [comment]
I agree. This project is excellent. Clean, straight forward, and works.  A++
